polkadot-js/alphanet-types.json

```diff
@@ -2,7 +2,6 @@
   "AccountId": "EthereumAccountId",
   "Address": "AccountId",
   "Balance": "u128",
-  "RefCount": "u8",
```

Comment:
@joelamouche Heads up for you. After this update gets merged, you will need to adjust the types that polkadot js Apps uses.

Reply:
Done in https://github.com/polkadot-js/apps/pull/4151